packages/chat-stateful-client/src/ChatContext.ts

```diff
@@ -19,6 +19,10 @@ import { _safeJSONStringify, toFlatCommunicationIdentifier } from '@internal/acs
 import { Constants } from './Constants';
 import { TypingIndicatorReceivedEvent } from '@azure/communication-chat';
 import { chatStatefulLogger } from './Logger';
+/* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
+import { CommunicationTokenCredential } from '@azure/communication-common';
```

Comment:
Could you update `import` to `import type` as we probably can import only type instead of the whole object here?

It helps bundlers to understand a little bit better what resources need to be included

Reply:
Done